packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -17,6 +19,15 @@ const generateAssetData = async () => {
     })
   )
 
+  const extendedERC20Tokens = await extendErc20()
+
+  const ethereumAsset = generatedAssetData.find(
+    (asset: Asset) => asset.caip19 === 'eip155:1/slip44:60'
+  )
+  if (ethereumAsset && extendedERC20Tokens?.length) {
+    ethereumAsset.tokens = flatten([ethereumAsset.tokens as TokenAsset[], extendedERC20Tokens])
```

Comment:
why are we casting here?

Reply:
My compiler was showing the tokens as any[]. I was trying to get it to recognize it as a TokenAsset. Wasn't necessary. Taken out.

Reply:
don't you want to do `flatten([ethereumAsset.tokens ?? [], extendedERC20Tokens]` because the compiler should already know it's an array of `TokenAsset` if it's defined?

Reply:
That makes sense. Done.